lib/plugins/sfDoctrinePlugin/lib/task/sfDoctrineBuildTask.class.php

```diff
@@ -45,13 +45,13 @@ protected function configure()
         $options = [
             new sfCommandOption('application', null, sfCommandOption::PARAMETER_OPTIONAL, 'The application name', true),
             new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED, 'The environment', 'dev'),
+            new sfCommandOption('all-classes', null, sfCommandOption::PARAMETER_NONE, 'Build all classes'),
         ];
 
         if (!sfProjectConfiguration::getActive()->isProduction()) {
```

Comment:
This has erroneously been returning `true` for a long time, even in production environments. When the method was updated to be more declarative, this flag suddenly sprang to life and blocked access to these tasks in production. Great!

The not-so-great part is that the php82 beta node also blocked the `--all-classes` task, which it needs for `flu.sh`. By moving the `--all-classes` task out of the `isProduction()` check, it becomes available again on the beta node.